Chrysalis-Papilio/Soen-490-Papilio-Android
# Papilio 97 set cost

# General info

**Issue number**: PAPILIO-97

**Task description**: 

 - add two new parameters to the activity creation : cost per individual and cost per group 

# Testing
To test this feature you have to:

1. open the app
2. login to your account 
3. go to the activity creation page
4. click on the "+" button 
5. fill out all the filed 
6. click the create activity and see if the new activity is in the database with  all the individual and group cost columns showing the correct number
